### PR TITLE
Implement the ping command for database.

### LINKED ISSUE #469: Implement Database.command, at least the ping command
My application health check is sending a ping command but I noticed command method is not implemented in Database, leading to command being interpreted as a Collection.

Maybe we could foresee a first implementation only handling 'ping' command and returning {'ok': 1.0} ?

### PR DESCRIPTION
**Pull request by pcorpet**
Fixes #469.

### CODE REVIEW

**Review by Colin-b — APPROVED** [review]
I tried both 'ping' as a string and {'ping': 1} as a dict, both command should work. This implementation is great, clever use of the in keyword to match both string and dict key commands :+1:

File: tests/test__database_api.py [hunk 1]
```diff
@@ -18,7 +18,11 @@ def test__get_collection_by_attribute_underscore(self):
         database['_users'].insert_one({'a': 1})
         self.assertEqual(1, database['_users'].find_one().get('a'))
 
+    def test__command_ping(self):
```
Comment (Colin-b) [inline tests/test__database_api.py:21]: Just a suggestion, you could maybe add an additional test case to ensure that command as a string works:
```suggestion
    def test__command_ping_as_str(self):
        database = mongomock.MongoClient().somedb
        self.assertEqual({'ok': 1}, database.command('ping'))

    def test__command_ping(self):
```

Comment (pcorpet) [reply tests/test__database_api.py:21]: Woops, you're right. I've also added a test for a string containing "ping" but not "ping".

**Comment by pcorpet** [comment]
:-) It was not on purpose. I had not plan for using it with a string and my implementation would also ping on command strings containing "ping" but not being "ping". Fixed.

**Pull request merged**